charmcraft/templates/init-simple/README.md.j2

```diff
@@ -19,6 +18,9 @@ TODO: Include a link to the default image your charm uses
 
 ## Contributing
 
-Please see the [Juju SDK docs](https://juju.is/docs/sdk) for guidelines
+<!-- TEMPLATE-TODO: Change this URL to be the correct Github path to CONTRIBUTING.md-->
```

Comment:
This file needs overhauling in line with the branch here: https://github.com/canonical/template-operator/blob/charmcraft-templates/default/README.md